### ISSUE TITLE
Disable motion & orientation access on iOS < 12.2

### ISSUE THREAD

**Issue by exxr** [opa334/SafariPlus#10]
How hard is it to implement this privacy feature in Safari before iOS 12.2 when Apple patched it? Can we disable sensors access?

**Comment by opa334** [comment]
Disabling it globally should be doable as long as there is an API that's used by Safari to supply the data, site specific would be pretty difficult.

How does Apple handle this? Is there any video of this in action?

**Comment by exxr** [comment]
Yeah since iOS 12.2 Apple has a toggle to enable/disable the DeviceMotionEvent and DeviceOrientationEvent events. See this article https://www.macrumors.com/2019/02/04/ios-12-2-safari-motion-orientation-access-toggle/

For more info on the privacy issues, check this out https://sensorid.cl.cam.ac.uk
